javascript/javascript2/week1/Javascript2Project/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Memory Card Game</title>
+  </head>
+  <body>
+    <section class="cards">
+      <div id="cards-grid" class="cards__box">
+        <div id="landing-text">
+          <h1>This is a memory Card Game</h1>
+          <h2>It may be easy, or not! Who knows?</h2>
+          <p>Press Start to chose difficulty</p>
+          <button onclick="choseDifficulty()">Start</button>
```

Comment:
Prefer binding events in your JS  file so everything is in one place. It is more manageable IMO